presto-hive/src/main/java/com/facebook/presto/hive/orc/HdfsOrcDataSource.java

```diff
@@ -74,7 +74,7 @@ protected void readInternal(long position, byte[] buffer, int bufferOffset, int
                 throw new PrestoException(HIVE_MISSING_DATA, message, e);
             }
             if (e instanceof IOException) {
-                throw new FileSystemCommunicationException(message, e);
```

Comment:
I think we need to keep `FileSystemCommunicationException`. Just move the definition here.